addons/calendar_sms/models/calendar.py

```diff
@@ -15,7 +15,7 @@ def _sms_get_default_partners(self):
         """ Method overridden from mail.thread (defined in the sms module).
             SMS text messages will be sent to attendees that haven't declined the event(s).
         """
-        return self.mapped('attendee_ids').filtered(lambda att: att.state != 'declined').mapped('partner_id')
+        return self.mapped('attendee_ids').filtered(lambda att: att.state != 'declined').mapped('partner_id').filtered('phone_sanitized')
```

Comment:
Hello. Why do you filter a first time on the state then another time on the phone_sanitized field. You could use a single filtered with both conditions.

Reply:
Of course, you're right. Thanks!